src/logcollector/logcollector.c

```diff
@@ -214,7 +214,7 @@ void LogCollectorStart()
                     logff[i].file = NULL;
                 }
                 logff[i].read = read_djbmultilog;
-            } else if (logff[i].logformat[0] >= '0' && logff[i].logformat[0] <= '9') {
+            } else if (strstr(logff[i].logformat, "multi-line:")) {
```

Comment:
Please consider using `strncmp()` to check that the string starts with "multi-line:".